core/alarm/alarm.go

```diff
@@ -191,12 +203,18 @@ func (as *alarmScheduler) Reset(alarmID string) {
 		alarm:   nil,
 		event:   cancel,
 	}
-
+	logIfChronologyAlarm(alarmID, "reset called")
 	as.event <- evt
 
 	as.Add(callback, duration, alarmID)
 }
 
+func logIfChronologyAlarm(alarmID string, event string) {
```

Comment:
maybe add a TODO to remove this after we find the root cause?

Reply:
done